### DISCUSSION TITLE
How do I add letters to the alphabet

### DISCUSSION THREAD

**Discussion by THEkatinamicrowave** [CodenameCrew/CodenameEngine#303] [Q&A]
This should work, right?
![alphabet-bold](https://github.com/FNF-CNE-Devs/CodenameEngine/assets/89645987/7301e26c-d1cf-47c1-9ffa-3968db7493bc)
```xml
...
	<SubTexture name="giggly little shit0000" x="771" y="215" width="88" height="68"/>
	<SubTexture name="😂0000" x="860" y="215" width="88" height="71"/>
</TextureAtlas>

```

**Comment by r6915ee** [comment]
> This should work, right? ![alphabet-bold](https://private-user-images.githubusercontent.com/89645987/332234910-7301e26c-d1cf-47c1-9ffa-3968db7493bc.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTYyNTkwNjMsIm5iZiI6MTcxNjI1ODc2MywicGF0aCI6Ii84OTY0NTk4Ny8zMzIyMzQ5MTAtNzMwMWUyNmMtZDFjZi00N2MxLTlmZmEtMzk2OGRiNzQ5M2JjLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA1MjElMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNTIxVDAyMzI0M1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTNiNTcwNWJhM2FiYmUwYzVlMWRlNWZhMTZjYTAzOWQ4ZjU5ZTUzMmExNDJhN2FhOTFkOWViZGUxYmI1NzAyMWYmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.eDxe3FglggVh5D8_mEmY2xhhJez6VtuRddli7f9Gdv8)
> 
> ```
> ...
> 	<SubTexture name="giggly little shit0000" x="771" y="215" width="88" height="68"/>
> 	<SubTexture name="😂0000" x="860" y="215" width="88" height="71"/>
> </TextureAtlas>
> ```

According to my knowledge, with this kind of format it might not work. So far I haven't figured out how to fix this, so for now some patience might be involved, sorry.

**Reply by THEkatinamicrowave** [reply]
alphbet-bold is a private image
![image](https://github.com/FNF-CNE-Devs/CodenameEngine/assets/89645987/5e4172f6-8521-4036-94fd-2486c91b6022)

**Reply by r6915ee** [reply]
What do you exactly mean, perhaps? If it's the quote then it's just the same image from above.

**Reply by THEkatinamicrowave** [reply]
oh ok then
